plugins/action/net_put.py

```diff
@@ -142,11 +142,10 @@ def _handle_existing_file(self, conn, source, dest, proto, timeout):
         filename = str(uuid.uuid4())
         tmp_source_file = os.path.join(cwd, filename)
         try:
-            conn.get_file(
+            conn.put_file(
```

Comment:
@KB-perByte I am trying to understand what _handle_existing_file  does ?
isn't it confirming the existence of the file we want to copy?
am I missing something here?

Reply:
Well for starters I feel there are multiple things wrong with the module.
The module never calls the put() API from the SCP object. 

The name of the method speaks about checking if the file exists or not, and hence get_file was used. Which again is a correct behavior, but incomplete in terms of operation. 
The PR is still expected to get a few more changes. This is there to check if put works or not.
and, it does. We have to redo a few things here to actually fix the problem.

Reply:
It gets more interesting, as parmiko works with or without the change.
I am trying to get more answers by reading the code in detail. I'll share by EOD